routers/comments.py

```
@@ -0,0 +1,161 @@
+    updated_comment = data.find_by_id("comments.json", comment_id)
+    return CommentResponse(
+        id=updated_comment["id"],
+        post_id=updated_comment["post_id"],
+        user_id=updated_comment["user_id"],
+        author_nickname=updated_comment["author_nickname"],
+        content=updated_comment["content"],
+        created_at=updated_comment["created_at"],
+        updated_at=updated_comment["updated_at"]
+    )
+@router.get("/me", response_model=List[CommentResponse], status_code=status.HTTP_200_OK)
+async def get_my_comments(current_user: dict = Depends(auth.get_current_user)):
```

Ruff (0.14.11)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)